modules/api/src/main/java/org/apache/ignite/table/KeyValueView.java

```diff
@@ -41,8 +41,8 @@
      * <p>Note: If the value mapper implies a value can be {@code null}, then a suitable method
      * {@link #getNullable(Transaction, Object)} must be used instead.
      *
-     * @param tx  The transaction or {@code null} to auto commit. 
-     * @param key A key which associated the value is to be returned. The key cannot be {@code null}.     
+     * @param tx  The transaction or {@code null} to auto commit.
+     * @param key A key which associated the value is to be returned. The key cannot be {@code null}.
      * @return Value or {@code null}, if it does not exist.
      * @throws IllegalStateException If value for the key exists, and it is {@code null}.
```

Comment:
IMHO, it should be a special exception instead of a just `IllegalStateException`. it is true for all methods which assume that value can't be `null`.